2_process/src/process_hru_sf.R

```diff
@@ -0,0 +1,23 @@
+topo_to_sf <- function(target_name, topo_fn, proj_start, proj_target) {
+  
+  hru_topo <- topojson_read(topo_fn)
+  hru_sf <- st_as_sf(hru_topo)
+  st_crs(hru_sf) <- proj_start
+  hru_sf_transf <- st_transform(hru_sf, proj_target)
+  
+  # Self-intersections were causing downstream problems with cropping and summarizing for a dissolve
+  hru_sf_valid <- st_make_valid(hru_sf_transf) # Needs pkg `lwgeom`
```

Comment:
There is code to parallelize this here if you need/want to: https://github.com/usgs-makerspace/wbeep-processing/blob/master/src/process_hru_shapes.R

Reply:
Made an issue for future improvements, #3